test/integration/lib/test-actions/open-payments.ts

```diff
@@ -320,8 +323,7 @@ async function createOutgoingPayment(
     },
     {
       walletAddress: senderWalletAddress.id,
-      metadata: {},
-      quoteId: quote.id
+      ...createArgs
```

Comment:
this one we should be careful about and put `walletAddress: senderWalletAddress.id` last: even though TS might be happy with it, you can still pass in a createArgs object with walletAddress defined at runtime, and it will override it

Reply:
fair enough - changed the order